plugin/pom.xml

```diff
@@ -79,6 +79,12 @@
           <artifactId>credentials</artifactId>
           <version>2.2.0</version>
           <scope>compile</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>io.jenkins.configuration-as-code</groupId>
+            <artifactId>parent</artifactId>
```

Comment:
I think this should be:
`configuration-as-code` not `parent`

apart from that it looks correct except for the indentation looks off